src/arch/xtos-wrapper/include/kernel.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I'd reword the commit to explain what the commit does. Once merged, people won't see the pull-request context and "first commit" won't make sense.

Reply:
Maybe refer to the place in source where the implementation is taken from and state this is implementing k_yield() Zephyr interface using XTOS.